### ISSUE TITLE
Substat Optimizer - No debug version

### ISSUE THREAD

**Issue by IvanHdzF** [genshinsim/gcsim#560]
From my understanding there's currently no way of disabling the debug log run when calling the sim, this makes run times take longer; originally this isn't a big issue per se when simulating normally, but it could take a bigger impact on the subs stat optimizer.

**Comment by jordanlovato** [comment]
@srliao I'm more interested in which aspects of this sim could be disabled during a substat optimization run. In theory I think we could turn off some of the logging operations, though this may impractical/impossible.

Turning off the debug log run is along that same vein, but small potatoes in the grand scheme of things.

**Comment by jordanlovato** [comment]
Note to self, I'm like 65% sure that we queue up all of our sim runners as goroutines and distribute substats after all goroutines have returned. I don't think there's any reason to wait on each run of the sim. I think currently the substat optimizer is psudeo-synchronous processing 10-20 runs of gcsim.

**Comment by jordanlovato** [comment]
Next idea, can we do small sim_iter runs during optimization passes to guide the larger sim_iter runs?

**Comment by jordanlovato** [comment]
> Staeld — Today at 9:06 PM
If I understood correctly, Raiden has a special case because she scales with ER, right? Shouldn't Mona also be considered or her ER scaling is not big enough to be a problem?

**Comment by srliao** [comment]
no longer relevant
